### PR TITLE
Paging package is added

### PR DESCRIPTION
**Pull request by shreeharsha-factly**

### CODE REVIEW

File: util/paging.go [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+import (
+	"net/url"
+	"strconv"
+)
+
+// Paging - pagination
+func Paging(query url.Values) (int, int) {
+
+	page, _ := strconv.Atoi(query.Get("page"))     // page number
```
Comment (dmonark) [inline util/paging.go:11]: what if page parameters is not there

File: util/paging.go [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+	perPage, _ := strconv.Atoi(query.Get("limit")) // perPage number
```
Comment (dmonark) [inline util/paging.go:12]: what if limit parameters is not there

File: util/paging.go (hunk 1 above)
Comment (shreeharsha-factly) [reply util/paging.go:11]: If page or limit is not there then default values are taken for limit & offset ( 5, 0)

**Review by dmonark — APPROVED** [review]
LGTM

**Pull request merged**